discovery-provider/src/queries/queries.py

```diff
@@ -165,6 +165,7 @@ def get_tracks_including_unlisted():
 
     db = get_db_read_replica()
     with db.scoped_session() as session:
+        # Create base query, filtering out stems
```

Comment:
@piazz is there supposed to be a filter here?

Reply:
Ah I removed the filter but forgot to take out the comment, will remove it! ty